README.md

```diff
@@ -5,11 +5,15 @@
 My Learning Analytics based on django framework
 
 ## Environment configuration
-There is some environment configuration and addtional files needed for this app to run. You can put this in a file called .env for testing. 
+There is some environment configuration for docker-compose on local host and addtional files needed for this app to run. Most of these files are in config directory.
 
 Configuration:
 
-Copy the file `.env.sample` to `.env` and fill in with the values for local testing. You may also provide a .env file if you have one.
+- Copy the file `.env.sample` to `.env` and fill in with the values for local testing. You may also provide a .env file if you have one.
+
+- You also need to copy the file config/env_sample.json to config/env.json. This is where most of the configuration is done now. 
+
+- There is a utility "env_to_json.py" to help convert your configuration if you were using this prior to JSON configuration. 
```

Comment:
may be helpful to include the following command line here, too:

python env_to_json.py > config/env.json

Reply:
Thanks, I'll add that to the README. There shouldn't be that many people that need this but it was useful for me to test a few ideas out before going down this path.